modules/nw-kuryr-cleanup.adoc

```diff
@@ -0,0 +1,217 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-kuryr-cleanup_{context}"]
+= Cleaning up resources after migration
+
+After migration is done, there are necessary steps to clean up resources created by Kuryr. From high level perspective it will include:
+
+* Removing Kuryr finalizers from resources
+* Removing Kuryr CRDs
+* Removing {rh-openstack-first} resources
```

Comment:
```suggestion
After migration completes, you can clean up resources that Kuryr created by:

* Removing Kuryr finalizers from resources
* Removing Kuryr CRDs
* Removing {rh-openstack-first} resources
```
What is "from resources" referring to here?

Reply:
by "finalizers from resources" here I have OCP resources like pods, services etc in mind.